grabbit/src/main/groovy/com/twcable/grabbit/client/batch/workflows/impl/DefaultWorkFlowManager.groovy

```diff
@@ -100,7 +100,7 @@ class DefaultWorkFlowManager implements WorkflowManager {
             return
         }
         if (configEntry.enabled == enable) {
-            log.error "Was expecting to change status of ${configId} to enabled=${enable}, but config entry was already of status enabled=${configEntry.enabled}"
+            log.warn "Was expecting to change status of ${configId} to enabled=${enable}, but config entry was already of status enabled=${configEntry.enabled}"
```

Comment:
Would setting `enabled` be considered an idempotent operation? If so, then this would be more of an `info` than `warn`.